src/GravityVisitor.h

```diff
@@ -52,11 +52,13 @@ class GravityVisitor {
 
   bool open(const SpatialNode<CentroidData>& source, SpatialNode<CentroidData>& target) {
     if (source.n_particles <= nMinParticleNode) return true;
-    Vector3D<Real> dr = source.data.centroid - target.data.centroid;
-    Real dsq = dr.lengthSquared();
-    if (theta * dsq < source.data.rsq) {
+    if (intersect(source.data.box, target.data.box.center(), source.data.rsq))
```

Comment:
I don't think you want to use 'source.data.rsq' here. There should still be a dependence on theta. See:

[https://github.com/N-BodyShop/changa_uw/blob/master/gravity.h#L613](url)

Reply:
This will work IF source.data.rsq is modified to include 1/theta.  But I don't see that happening in this PR.